extension/js/common/core/const.ts

```diff
@@ -25,7 +25,7 @@ export const GMAIL_RECOVERY_EMAIL_SUBJECTS = [
   'Your FlowCrypt Backup',
   'Your CryptUp Backup',
   'All you need to know about CryptUP (contains a backup)',
-  'CryptUP Account Backup',
+  'CryptUP Account Backup'
```

Comment:
I think the separator on every line is better, for languages that support it. It would be best to enforce that through linting, if possible.

Then you can add or remove lines without having to modify other lines that you didn't mean to add/remove (by adding/removing comma there), which keeps git history cleaner.

Reply:
Agree, `trailingComma` rule was enabled by default, but I disabled it as such separators were rarely used in FlowCrypt codebase. Will enable it back